src/renderer/services/eth2/client/prysm/CgPrysmEth2BeaconBlocksApi.ts

```diff
@@ -0,0 +1,29 @@
+import {CgEth2BeaconBlocksApi} from "../eth2ApiClient/cgEth2BeaconBlocksApi";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {HttpClient} from "../../../api";
+import {SignedBeaconBlock} from "@chainsafe/lodestar-types";
+import {matomo} from "../../../tracking";
+import {SignedBeaconBlockData} from "./map.types";
+import {mapProduceBlockDataToPrysmProduceBlock} from "./mapProduceBlockDataToPrysmProduceBlock";
+
+export class CgPrysmEth2BeaconBlocksApi extends CgEth2BeaconBlocksApi {
+    public constructor(config: IBeaconConfig, httpClient: HttpClient) {
+        super(config, httpClient);
+    }
+
+    public publishBlock = async (block: SignedBeaconBlock): Promise<void> => {
+        const data = (this.config.types.SignedBeaconBlock.toJson(block, {
+            case: "snake",
+        }) as unknown) as SignedBeaconBlockData;
+        console.log(JSON.stringify(data, undefined, 2));
+        console.log(JSON.stringify(mapProduceBlockDataToPrysmProduceBlock(data), undefined, 2));
+        await this.httpClient.post("/eth/v1alpha1/validator/block", mapProduceBlockDataToPrysmProduceBlock(data));
```

Comment:
throws error
```json
{
  "code": 13,
  "message": "Could not process beacon block: could not process block: signature in block failed to verify",
  "details":[]
}
```
CG returns 
```json
{
  "message": {
    "slot": "29778",
    "proposer_index": "7",
    "parent_root": "0xaa865ba6ddb14eef45ce42718fad9bda27e4b9cda97c8be76a92379f4474d811",
    "state_root": "0xe4b497e05c129ad68e064f9c7fd5c891dac6f30abcaa282d62552d8b22814761",
    "body": {
      "randao_reveal": "0x879c1c47ec423472a6d04dad252bf21d4bb81053d9e63a82d0cc800bf58e8e9ea2c6f515dfd9ad1280de75ee5a8c9468010c1b256c2b8b95c2d88098e77a4909875822eb479d84478bb1374e449e32954d1d88dc8397c297c55b186735bef14c",
      "eth1_data": {
        "deposit_root": "0x52fcacfa96a46851ef0752505177ba91e000eb73d41b05be6e64c56320f8af7f",
        "deposit_count": "16",
        "block_hash": "0x764e761c474b3a3ddef4ce79e291d59a8f730056461b8c6edfce7c2d7bca443a"
      },
      "graffiti": "0x436861696e477561726469616e00000000000000000000000000000000000000",
      "proposer_slashings": [],
      "attester_slashings": [],
      "attestations": [
        {
          "aggregation_bits": "0x03",
          "data": {
            "slot": "29777",
            "index": "0",
            "beacon_block_root": "0xaa865ba6ddb14eef45ce42718fad9bda27e4b9cda97c8be76a92379f4474d811",
            "source": {
              "epoch": "929",
              "root": "0xd9791025b37c10e527bcddbe281f671a476bcdd1466383ff049282dad78ac2a6"
            },
            "target": {
              "epoch": "930",
              "root": "0x4bc18d565c06e43ddb69e81a6547faa52df85423e6e365c15ee2633b70ca393c"
            }
          },
          "signature": "0x8b538025d73f93ba388220740811f7b084831c7a92a053da407bcb53fc607c6a5cc19bf6a3eb6a0ea090bfb9abe7c7ea1812e23e05e3e784d35ab785fca969de5886a1cd46ee87fdd2467842ccc010afb95aaf098f687459b4625ab867a1e546"
        }
      ],
      "deposits": [],
      "voluntary_exits": []
    }
  },
  "signature": "0x9157114008e8589aaabb423b18e0b92336f13d9adf4a3a732967e5234a01ab6697ccf1d26d8e334b1c92d08efd7afe63146db9a76309ce06d86d39f4c4be7fde4448311db0fdf900d1398bbcc95d6fddb18372067b3fde53ffb3976064c439f8"
}
```
and mapping (from hex to base) results
```json
{
  "block": {
    "slot": "29778",
    "proposer_index": "7",
    "parent_root": "qoZbpt2xTu9FzkJxj62b2ifkuc2pfIvnapI3n0R02BE=",
    "state_root": "5LSX4FwSmtaOBk+cf9XIkdrG8wq8qigtYlUtiyKBR2E=",
    "body": {
      "randao_reveal": "h5wcR+xCNHKm0E2tJSvyHUu4EFPZ5jqC0MyAC/WOjp6ixvUV39mtEoDede5ajJRoAQwbJWwri5XC2ICY53pJCYdYIutHnYRHi7E3TkSeMpVNHYjcg5fCl8VbGGc1vvFM",
      "eth1_data": {
        "deposit_root": "Uvys+pakaFHvB1JQUXe6keAA63PUGwW+bmTFYyD4r38=",
        "deposit_count": "16",
        "block_hash": "dk52HEdLOj3e9M554pHVmo9zAFZGG4xu3858LXvKRDo="
      },
      "graffiti": "Q2hhaW5HdWFyZGlhbgAAAAAAAAAAAAAAAAAAAAAAAAA=",
      "proposer_slashings": [],
      "attester_slashings": [],
      "attestations": [
        {
          "aggregation_bits": "Aw==",
          "data": {
            "slot": "29777",
            "committee_index": "0",
            "beacon_block_root": "qoZbpt2xTu9FzkJxj62b2ifkuc2pfIvnapI3n0R02BE=",
            "source": {
              "epoch": "929",
              "root": "2XkQJbN8EOUnvN2+KB9nGkdrzdFGY4P/BJKC2teKwqY="
            },
            "target": {
              "epoch": "930",
              "root": "S8GNVlwG5D3baegaZUf6pS34VCPm42XBXuJjO3DKOTw="
            }
          },
          "signature": "i1OAJdc/k7o4giB0CBH3sISDHHqSoFPaQHvLU/xgfGpcwZv2o+tqDqCQv7mr58fqGBLiPgXj54TTWreF/Klp3liGoc1G7of90kZ4QszAEK+5Wq8Jj2h0WbRiWrhnoeVG"
        }
      ],
      "deposits": [],
      "voluntary_exits": []
    }
  },
  "signature": "kVcRQAjoWJqqu0I7GOC5IzbxPZrfSjpzKWflI0oBq2aXzPHSbY4zSxyS0I79ev5jFG25p2MJzgbYbTn0xL5/3kRIMR2w/fkA0TmLvMldb92xg3IGez/eU/+zl2BkxDn4"
}
```

Reply:
:thinking:  
https://github.com/prysmaticlabs/ethereumapis/blob/9dd2f03bb0060535c65b8acee4e4b2b7fccf3b77/eth/v1alpha1/validator.proto#L67

Reply:
wrong fork version -.-